components/TopNav.vue

```diff
@@ -6,35 +6,36 @@
                     src="../assets/images/logo-32-px-color.svg" />Find a Doc
             </NuxtLink>
         </div>
-        <div>
+        <div >
             <NuxtLink to="/about">
                 About
             </NuxtLink>
-            <div class="cursor-pointer md:hidden">
-                <img src="../assets/images/hamburgerMenu.svg" alt="hamburger menu" />
+            <div  class="cursor-pointer md:hidden " >
```

Comment:
looks like you need to run your linter. Try running `yarn lint --fix`

Reply:
Ran linter and fixed issue with spacing